lib/kubernetes-deploy/runner.rb

```diff
@@ -37,6 +38,7 @@ class Runner
     PROTECTED_NAMESPACES = %w(
       default
       kube-system
+      kube-public
```

Comment:
er, not really. It's a new system ns in 1.6. I can move this commit out if you prefer.